index.html

```diff
@@ -2318,7 +2318,17 @@ <h3>HTML Element Role Mappings</h3>
               <td class="atk"><div class="general">Use WAI-ARIA mapping</div></td>
               <td class="ax"><div class="general">Use WAI-ARIA mapping</div></td>
               <!-- <td class="naming"></td> -->
-              <td class="comments"></td>
+              <td class="comments">
+                <div>
+                  If specified, the default subtree of a `meter` element maps to the 
+                  <a class="core-mapping" href="#ariaValueText">`aria-valuetext`</a>. 
```

Comment:
I don't understand what "default subtree of [an element] maps to [an attribute]"

Reply:
the intent in this instance is that `<meter>2cm</meter>` the '2cm' fallback/default value should be treated as if it was specified via aria-valuetext=2cm

Reply:
Okay then maybe this?
```suggestion
                  If the `meter` element includes [text content? innerText? computed `nameFrom:content`], user agents SHOULD expose that as the value of the <a class="core-mapping" href="#ariaValueText">`aria-valuetext`</a> mapping. 
```

Reply:
that works for me.  i'll commit the change.  thank you